pkg/app/server/httpapi/httpapimetrics/handler_test.go

```diff
@@ -0,0 +1,144 @@
+package httpapimetrics
+
+import (
+	"testing"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLabels(t *testing.T) {
+
```

Comment:
Please make those tests run in parallel. Please check other test files for reference 😄